site2/docs/about.md

```diff
@@ -27,9 +27,9 @@ Select one of the content blocks below to begin your Pulsar journey. If you ...
 <BlockLinks>
     <BlockLink title="About Pulsar" url="/docs/next/concepts-overview/" />
     <BlockLink title="Get Started" url="/docs/next/getting-started-home/" />
-    <BlockLink title="Install, Deploy, Upgrade" url="/docs/next/deploy-aws/" />
+    <BlockLink title="Install, Deploy, Upgrade" url="/docs/next/install-deploy-upgrade-landing/" />
     <BlockLink title="Pulsar for Developers" url="/docs/next/developers-landing/" />
-    <BlockLink title="How To" url="/docs/next/functions-develop/" />
+    <BlockLink title="How To" url="/docs/next/how-to/" />
```

Comment:
Since you've changed the topic ID to `how-to-landing` in the target file, it needs an update here to keep consistent with the topic ID.